tests/js/HappinessLintBearTest.py

```diff
@@ -17,3 +23,19 @@
 HappinessLintBearTest = verify_local_bear(HappinessLintBear,
                                           valid_files=(good_file,),
                                           invalid_files=(bad_file,))
+
+
+@generate_skip_decorator(HappinessLintBear)
+class HappinessLintBearConfigTest(LocalBearTestHelper):
+
+    VALUE_ERR_RE = ('Set "use_spaces=False" for HappinessLintBear '
+                    'as "use_spaces=Ture" is incompatible '
+                    'for HappinessLintBear.')
+
+    def test_validate_use_spaces(self):
+        section = Section('name')
+        section.append(Setting('use_spaces', True))
+        bear = HappinessLintBear(section, Queue())
+
+        with self.assertRaisesRegex(AssertionError, self.VALUE_ERR_RE):
```

Comment:
Not a `ValueError`? As this is thrown inside the bear. Or does coala wrap this exception up in an `AssertionError`?

Reply:
yup coala wraps it in AssertionError